keras_trainer/data_generators.py

```diff
@@ -138,6 +141,23 @@ def random_crop_parameters(img, crop_size, crop_mode='size'):
 
         return x, y, x + tw, y + th
 
+    @staticmethod
+    def apply_custom_crop(image, crop_coordinates):
+        """
+        The format of the image cropping coordinates should be [x, y, width, height] and PIL cropping format is
```

Comment:
Maybe to better reference our expected arguments:
```suggestion
        The format of the image `crop_coordinates` should be [x, y, width, height] and PIL cropping format is
```